modules/ROOT/pages/dashboard-custom-config-graph.adoc

```diff
@@ -96,23 +97,24 @@ After you complete your configuration, you can click Save Changes. To exit the c
 [[alerts_config]]
 == Alerts (Configuration, Notifications, History)
 
-//*https://www.mulesoft.com/anypoint-pricing[Titanium Subscription Required]*
-*Titanium Subscription Required*
+[NOTE]
+A Titanium subscription is required to use this feature. 
 
-In the Alerts tab, you can set up alerts that trigger when JVM memory utilization is too high.
+In the *Alerts* tab, set up alerts that trigger when JVM memory utilization is too high.
 
 include::{partialsdir}/include-alerts-config.adoc[]
 
 //END ALERTS
 
 == Advanced (Legends, Thresholds, Stacking, Hovering, Time Shift Settings)
-You can add and configure legends, thresholds, and other features for your graph.
+
+Add and configure legends, thresholds, and other features for your graph.
 
 ////
 TODO_REMOVED FOR NOW PER PABLO:
 
 === Links (Advanced Setting)
-In the Advanced configuration tab, you can configure links that appear in the dropdown menu in a for a graph in a custom dashboard, for example:
+In the Advanced configuration tab, you can configure links that appear in the dropdown menu for a graph in a custom dashboard, for example:
```

Comment:
I don't understand what this means. "for a graph"? since when do graphs have dropdown menus, and why would there be links in them? Maybe write a sentence of context? I just don't get what's going on here.
I'd remove "you can" everywhere. 
"configure the links" (i.e. links always appear)?

Reply:
how is a graph in a custom dashboard different from a graph anywhere else?

Reply:
This section is commented out. It doesn't show up in the content.